cmd/tetra/debug/progs.go

```diff
@@ -137,7 +155,7 @@ Examples:
 
 	flags := cmd.Flags()
 	flags.BoolVar(&cfg.all, "all", false, "Get all programs")
-	flags.StringVar(&cfg.lib, "bpf-lib", "bpf/objs/", "Location of Tetragon libs (btf and bpf files)")
+	flags.StringVar(&cfg.lib, "bpf-lib", "?", "Location of Tetragon libs, btf and bpf files, '?' to auto detect")
```

Comment:
Any reason of why not using the empty string?
Not a blocker, just curious.

Reply:
heh, I thought that's what you asked for :-) but it's -- ""? --  I need eye checkup, will change